Configuration/src/main/java/io/deephaven/configuration/Configuration.java

```diff
@@ -36,11 +36,18 @@ public class Configuration extends PropertyFile {
     private Collection<String> contextKeys = Collections.emptySet();
 
     /**
-     * The default configuration implementation loading the property file from the default property file.
+     * The default Configuration implementation, which loads properties from the default property file in
+     * addition to System properties.
      */
     private static class DefaultConfiguration extends Configuration {
```

Comment:
I would remove the `Configuration` suffixed from the nested impls. `Configuration.Default` sounds better than `Configuration.DefaultConfiguration`. Ditto `Configuration.Named`.